# This represents an ordered sequence of 2-D grid maps, where each 
# ProbabilityGrid is generated for a certain timestep (in the future). 
# Each grid cell represents the probability of occupancy

ProbabilityGrid[] gridarray
